packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -699,8 +701,8 @@ export function buildWebpackBrowser(
           event =>
             ({
               ...event,
-              // If we use differential loading, both configs have the same outputs
-              outputPath: baseOutputPath,
+              baseOutputPath: baseOutputPath,
+              outputPaths: outputPaths || baseOutputPath,
```

Comment:
```suggestion
              outputPaths: outputPaths || [baseOutputPath],
```

Reply:
I think adding back the `outputPath` option as deprecated and make it equal to the first value would be useful as well.  It would remove the need to change a large volume of tests as well as prevent a breaking change for anyone currently using the builder directly.

Reply:
PTAL